src/EnergyPlus/SolarShading.cc

```diff
@@ -7194,6 +7194,10 @@ namespace SolarShading {
 			} else {
 				SUN3( DayOfYear, AvgSinSolarDeclin, AvgEqOfTime );
 				AvgCosSolarDeclin = std::sqrt( 1.0 - pow_2( AvgSinSolarDeclin ) );
+				// trigger display of progress in the simulation every two weeks
```

Comment:
@Myoldmopar I don't think there are any side affects It gets used at:

https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/SimulationManager.cc#L548

and

https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/HVACSizingSimulationManager.cc#L349

But just to trigger the output window display of "Continuing Simulation at...". It time that is displayed it is set to false.